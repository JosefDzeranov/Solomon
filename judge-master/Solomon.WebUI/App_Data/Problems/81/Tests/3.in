15
53 8 99 1474 85 3887 53 856 917 71 422 33 2 378 5