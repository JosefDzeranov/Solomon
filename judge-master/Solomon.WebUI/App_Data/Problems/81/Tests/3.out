59 8 101 1474 89 3887 59 856 917 73 422 37 2 378 5 